cloudify_agent/tests/api/pm/test_base.py

```diff
@@ -32,7 +32,9 @@ def setUp(self):
         self.daemon = Daemon(
             manager_ip='manager_ip',
             queue='queue',
-            name='name'
+            name='name',
+            broker_user='guest',
```

Comment:
no real need to pass this in all the tests here as this is the default value in base.py anyway